src/api_client.coffee

```diff
@@ -166,19 +166,19 @@ class APIClient extends ironCore.Client
       parseResponseBind(error, response, body, cb)
     )
 
-  messagesTouch: (queue_name, message_id, cb) ->
-    body = _.pick(options, 'reservation_id')
+  messagesTouch: (queue_name, message_id, reservation_id, cb) ->
```

Comment:
name of the function must be `messageTouch`, because multiple messages touch is not supported. Also, this method must support `options`. Currently, IronMQ v3 supports only `timeout` option.